src/frontend-common/imgui_manager.cpp

```diff
@@ -488,14 +488,13 @@ bool ImGuiManager::AddImGuiFonts(bool fullscreen_fonts)
 
   if (fullscreen_fonts)
   {
-    // Don't add full range for FSUI, we probably don't have the texture space, and it's not translated yet anyway.
```

Comment:
The comment here illustrates the issue, especially at 4K display resolutions, the font texture gets way too large to include all font ranges. Hence why only the normal-sized font (used for OSD messages) uses the full range.

Reply:
Thanks for your comment.
I see what is the problem... so for ja/zh-cn, is it better not to add translation for these Achievement Summary / Notifications ..? (to avoid showing ???? and instead to show original English messages)

If I could generate custom Glyph Ranges based on the translation text, using AddText() of the ImFontGlyphRangesBuilder  like below,
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md#using-custom-glyph-ranges
I guess we can solve that large texture issue, but unfortunately current QTranlator seems not providing functions to retrieve translated texts (messages).

How do you think if I create small offline tool which output ImVector<ImWchar> ranges based on *.ts files, and embed that output around below code in qthost.cpp: InstallTranslator() ?

`
  if (language == QStringLiteral("ja"))
  {
    ImGuiManager::SetFontPath(GetFontPath("msgothic.ttc"));
    ImGuiManager::SetFontRange(ImGui::GetIO().Fonts->GetGlyphRangesJapanese());
  }
`